### PR TITLE
Database certification - criteria, process and request form

### PR DESCRIPTION
**Pull request by jonnyrobots**
+ Adds menu link and new page for database "certification" (criteria, process and request form) 
+ Updates the existing "adding support" page to reflect wording change to "adding compatibility" 
+ Add's CTA's to the "support levels" page for steps towards guaranteed support
+ Orders "Supported databases" menu alphabetically

### CODE REVIEW

File: documentation/learnmore/getting-certified.md [hunk 1]
```diff
@@ -4,4 +4,58 @@ menu: gettingCertified
...
+- You are happy that Flyway’s prescribed migration process matches best-practice for your database, or are able to provide specific guidance in the context of using Flyway with your RDBMs. For example, some Cloud-native RDBMSs with auto-scaling features like geo-replication require the database change-frequency to be limited.    
+- You are able to provide documentation to help us understand any nuances specific to your database engine that we need to make Flyway users aware of. For example, special syntax.
+- You are able to support us in developing a comprehensive test suite for your database. For example, we require a Dockerized version of your database – or, for Cloud-native databases, an free instance or emulator (with parity) – for us to develop and run our tests against.
```
Comment (madninjaskillz) [inline documentation/learnmore/getting-certified.md]: `an free` - should be `a free`

File: documentation/learnmore/getting-certified.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+One of the things that sets Flyway apart is its extensive and comprehensive approach to testing. Today we have well over 3000 individual tests. These tests ensure that Flyway is dependable in the most demanding infrastructure environments.
+
+Each new database deserves the same degree of testing. As you can imagine, this adds some overhead to Flyway's core development team.
+
+Our relationships with database vendors is an important part of being a stable product, and it's a relationship that also benefits vendors. Today, the majority of databases that have grown in popularity operate on some form of the open-core model, providing paid-for editions of their technology in much the same way as Flyway.
+
+There are [339 database engines](https://db-engines.com/en/ranking) in the world, offering a lot of choice for developers, their specific needs and context of application. When we add support into Flyway for a new database, we strengthen the ecosystems of those vendors and their databases.
```
Comment (juliahayward) [inline documentation/learnmore/getting-certified.md]: There are almost certainly way more than 339, but certainly 339 _serious_ ones...

File: documentation/learnmore/getting-certified.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+### Criteria
+An RDBMS will be considered for Certification when we (the Flyway core team) are satisfied that:
+
+**There is user demand, and value.**
+
+- You can demonstrate that there is end-user (and/or customer) demand for Flyway to work with your RDBMS. This should be backed up with example qualitative data (eg. conversations with existing users or customers), and/or quantitative data (eg. number of “up-votes” on the GitHub issue, or position in the db-engines rankings table).  
+
+**There is technical compatibility.**
+
+- You can demonstrate your RDBMS is technically compatible with Flyway having forked Flyway and followed the Adding Compatibility process. Note that while it’s technically possible, we don’t currently support NoSQL databases due to the degree of architectural change required to Flyway core in order to support them. However, we may do so in the future. Any contributed code requires the author to have signed out CLA.
```
Comment (juliahayward) [inline documentation/learnmore/getting-certified.md]: "our" (penultimate word)

Comment (jonnyrobots) [reply documentation/learnmore/getting-certified.md]: fixed in latest commit

File: documentation/learnmore/getting-certified.md (hunk 2 above)
Comment (jonnyrobots) [reply documentation/learnmore/getting-certified.md]: simplified in latest commit

**Review by juliahayward — APPROVED** [review]
Looks good to me!

**Review by DoodleBobBuffPants — APPROVED** [review]

**Pull request merged**